src/context.h

```diff
@@ -45,7 +45,17 @@ namespace librealsense
     public:
         virtual std::shared_ptr<device_interface> create_device() const
         {
-            return create(_ctx, true);
+            auto device = create( _ctx, true );
+
+#ifdef BUILD_AUS
```

Comment:
`librealsense::aus_on_device_changed(device)`

1 line injection 

inside you can use the 
#ifdef BUILD_AUS

better not to use it all over..

Reply:
fixed